alphadia/fdrx/models/two_step_classifier.py

```diff
@@ -83,52 +90,94 @@ def fit_predict(
             DataFrame containing predictions and q-values
 
         """
+        logger.info("=== Starting training of TwoStepClassifier ===")
+
         df = self._preprocess_data(df, x_cols)
         best_result = None
-        best_precursor_count = -1
+
+        previous_target_count_after_first_clf = -1
```

Comment:
can you add a comment that they will track precursor and at which FDR (variable name)